src/site/apt/non-javac-compilers.apt.vm

```diff
@@ -46,7 +48,7 @@ Using Non-Javac Compilers
 
   * <<<eclipse>>> with artifactId <<<plexus-compiler-eclipse>>>.
 
-  * <<<jikes>>> with artifactId <<<plexus-compiler-jikes>>>.
+  * <<<javac-with-errorprone>>>  with artifactId <<<plexus-compiler-javac-errorprone>>>.
```

Comment:
javac-with-errorprone requires JDK 11+. Do you think this needs a special explanation?
https://codehaus-plexus.github.io/plexus-compiler/plexus-compilers/plexus-compiler-javac-errorprone/

Reply:
@CrazyHZM - requirements was added